pkgs/by-name/po/postlight-parser/package.nix

```diff
@@ -0,0 +1,33 @@
+{ lib
+, stdenv
+, mkYarnPackage
+, fetchFromGitHub
+, fetchYarnDeps
+}:
+
+mkYarnPackage rec {
+  pname = "postlight-parser";
+  version = "2.2.3";
+
+  src = fetchFromGitHub {
+    owner = "postlight";
+    repo = "parser";
+    rev = "v${version}";
+    hash = "sha256-k6m95FHeJ+iiWSeY++1zds/bo1RtNXbnv2spaY/M+L0=";
+  };
+
+  packageJSON = "${src}/package.json";
```

Comment:
This does an import from derivation. You could use `stdenv.mkDerivation` instead or vendor the file.

Reply:
I don't understand this part. `src` is a derivation. This is actually [the default](https://github.com/NixOS/nixpkgs/blob/98a0c372a314729fa4d811063ea2ad7f230b2f9b/pkgs/development/tools/yarn2nix-moretea/yarn2nix/default.nix#L252) I forgot to remove. Why would I need to do anything extra here?

Reply:
Because `mkYarnPackage` reads version, name, etc. from package.json. Since that's done at eval time it's IFD.